kotlinx.fuzz.api/src/main/kotlin/kotlinx/fuzz/config/GlobalConfig.kt

```diff
@@ -71,6 +72,12 @@ class GlobalConfigImpl internal constructor(builder: KFuzzConfigBuilder) : Globa
         validate = { require(it > 0) { "'threads' must be positive" } },
         default = max(1, Runtime.getRuntime().availableProcessors() / 2),
     )
+    override var supportJazzerTargets: Boolean by builder.KFuzzPropProvider(
+        nameSuffix = "supportJazzerTargets",
+        intoString = { it.toString() },
+        fromString = { it.toBooleanStrict() },
+        default = false,
```

Comment:
It seems more logical that we have it `true` by default

Reply:
I don't agree with this, since it's more of a legacy feature